src/apps/chifra/pkg/config/global_config.go

```diff
@@ -12,7 +12,14 @@
  *-------------------------------------------------------------------------------------------*/
```

Comment:
Can we please rename this file `trueblocks.go`? In that way, when I look in this folder, each file will be named the same as it's subcommand route except for this one. And the names of these go files will reflect the actual names of the toml files they read.

Reply:
Definitely, good idea